labs/KeyWrapping/KeyWrapping.md

```diff
@@ -177,19 +177,19 @@ The document indicates that `KW-AE` depends on a *wrapping function*
 which we will have to model in our formal specification:
 
   * A **Key Encryption Key (KEK)** `K` and
-  * A **designated cipher function** `CIPHk`, which operates on 128-bit blocks
+  * A **designated cipher function** CIPH<sub>K</sub>, which operates on 128-bit blocks
```

Comment:
These are backticked because they refer to the Cryptol function with the same name. I’d keep the backtick version